week2/project/ecommerce/src/App.js

```diff
@@ -0,0 +1,23 @@
+import './App.css';
+import { Routes, Route } from 'react-router-dom'
+import ProductsPage from './pages/ProductsPage/ProductsPage'
+import DetailProductPage from './pages/DetailProductPage/DetailProductPage'
+import NotFoundPage from './pages/NotFoundPage/NotFoundPage';
+import HomePage from './pages/HomePage/HomePage';
+
+
+
+function App() {
+
+   return (
+      <div className="App">
+         <Routes>
+            <Route path='/' element={<HomePage />} />
```

Comment:
I checked the `HomePage` component and it's empty then if you don't need the user to use the endpoint of `/` you can modify this line to be 
```js
        <Route path="/" element={<Navigate to="/products" />} />
```

and don't forget to import `Navigate` from react router dom
✏️need to modify✏️

Reply:
I changed the line.